From 9d12fa7996115ce8053da57b9d69b11eab562174 Mon Sep 17 00:00:00 2001
From: arter97 <qkrwngud825@gmail.com>
Date: Sun, 14 Jun 2015 19:37:45 +0900
Subject: [PATCH] adreno_idler: fix typos :)

Signed-off-by: arter97 <qkrwngud825@gmail.com>
---
 drivers/devfreq/adreno_idler.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/devfreq/adreno_idler.c b/drivers/devfreq/adreno_idler.c
index bdfad4da2d1..0e6cd27b1b1 100644
--- a/drivers/devfreq/adreno_idler.c
+++ b/drivers/devfreq/adreno_idler.c
@@ -51,8 +51,8 @@ static unsigned int idlewait = 20;
 module_param_named(adreno_idler_idlewait, idlewait, uint, 0664);
 
 /* Taken from ondemand */
-static unsigned int downdifferenctial = 20;
-module_param_named(adreno_idler_downdifferenctial, downdifferenctial, uint, 0664);
+static unsigned int downdifferential = 20;
+module_param_named(adreno_idler_downdifferential, downdifferential, uint, 0664);
 
 /* Master switch to activate the whole routine */
 static bool adreno_idler_active = true;
@@ -75,7 +75,7 @@ int adreno_idler(struct devfreq_dev_status stats, struct devfreq *devfreq,
 			return 1;
 		}
 		if (idlecount >= idlewait &&
-		    stats.busy_time * 100 < stats.total_time * downdifferenctial) {
+		    stats.busy_time * 100 < stats.total_time * downdifferential) {
 			/* We are idle for (idlewait + 1)'th time! Ramp down the frequency now. */
 			*freq = devfreq->profile->freq_table[devfreq->profile->max_state - 1];
 			idlecount--;
